src/main/java/com/faforever/client/clan/Clan.java

```diff
@@ -43,14 +43,15 @@ public Clan() {
 
   public static Clan fromDto(com.faforever.client.api.dto.Clan dto) {
     Clan clan = new Clan();
-    clan.setId(dto.getId());
+    clan.setId(Integer.parseInt(dto.getId()));
```

Comment:
Does anything prevent us from making this clan bean's ID as String as well? Otherwise, this is rather a gamble.